agenda/presentation/src/main/java/com/gkp/agenda/presentation/util/Util.kt

```diff
@@ -0,0 +1,46 @@
+package com.gkp.agenda.presentation.util
+
+import java.time.LocalDate
+private const val CURRENT_AND_NEXT_SIX_DATES = 7
+
+fun getFullNameInitials(fullName: String): String {
+    val names = fullName.split(" ")
+    val initialsBuilder = StringBuilder()
+    if (names.size == 1) {
+        initialsBuilder.append(names[0].take(2).uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size == 2) {
+        initialsBuilder.append("${names[0].first()}${names[1].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size >= 3) {
+        initialsBuilder.append("${names[0].first()}${names[2].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
```

Comment:
Even though rare, a name with two middle names would not be considered, since `Philipp Middle1 Middle2 Lackner` would result in the initials `PM` here